pages/admin/AdminHandler.php

```diff
@@ -247,54 +247,27 @@ public function settings($args, $request)
     /**
      * Business logic for site settings single/multiple contexts availability
      *
-     * @param PKPRequest $request
-     *
-     * @return array [siteComponent, availability (bool)]
+     * @return array<string,bool> List of tabs, where the key is the tab name and the value its availability
      */
-    private function siteSettingsAvailability($request)
+    private function siteSettingsAvailability(): array
     {
-        $tabsSingleContextAvailability = [
-            'siteSetup',
-            'languages',
-            'bulkEmails',
-            'statistics',
-        ];
-
-        $tabs = [
-            'siteSetup',
-            'siteAppearance',
-            'sitePlugins',
-            'siteConfig',
-            'siteInfo',
-            'languages',
-            'navigationMenus',
-            'highlights',
-            'bulkEmails',
-            'siteTheme',
-            'siteAppearanceSetup',
-            'statistics',
-            'announcements',
+        // The multi context UI is also displayed when the journal has no contexts
+        $isMultiContextSite = Services::get('context')->getCount() === 1;
```

Comment:
It should be `!==`, right? :sweat_smile:

Reply:
Definitely 😆
I'll use the opportunity to debug why the upload at the `NativeXmlImportExportSubmission` is failing.